packages/styles/scss/css-variables/variables.scss

```diff
@@ -162,6 +163,14 @@
 
     // Spacing
     @include spacing.emit-spacing-custom-properties();
+
+    // Standard colors
+    @include light-dark('color-red', rgb-color('red', '500'), rgb-color('red', '600'));
+    @include light-dark('color-orange', rgb-color('orange', '500'), rgb-color('orange', '600'));
+    @include light-dark('color-cyan', rgb-color('cyan', '500'), rgb-color('cyan', '600'));
+    @include light-dark('color-sky', rgb-color('sky', '500'), rgb-color('sky', '600'));
+    @include light-dark('color-yellow', rgb-color('yellow', '500'), rgb-color('yellow', '600'));
+    @include light-dark('color-amber', rgb-color('amber', '500'), rgb-color('amber', '600'));
```

Comment:
Not sure if you actually need these defined here.

If you look here https://github.com/NorskHelsenett/ror-web/blob/main/apps/web/src/styles/tailwind.css#L1

You can see that we import all the harmony colors, they are also compatible with tailwind v4 which means that you can use all the same tailwind utility classes to reference these colors, e.g. `bg-red-500`. At least in the web app.

If you want to export the foundational colors from the styles package I would do so with all the colors, for an outsider it's hard to see why there are only a subset of colors available.